### PR TITLE
Namespaces ui: and cls: attributes with nx-

### PR DESCRIPTION
**Pull request by djsauble**
https://issues.sonatype.org/browse/NEXUS-7660

Related to: https://github.com/sonatype/nexus-pro/pull/470

### CODE REVIEW

File: plugins/rapture/nexus-rapture-baseapp/src/main/baseapp/packages/rapture-theme/sass/src/Ext/button/Button.scss [hunk 1]
```diff
@@ -43,12 +43,12 @@
   $inner-opacity-disabled: 1
 );
 
-.x-btn-drilldown-large {
+.x-btn-ux-drilldown-large {
```
Comment (adreghiciu) [inline plugins/rapture/nexus-rapture-baseapp/src/main/baseapp/packages/rapture-theme/sass/src/Ext/button/Button.scss]: ux?

File: plugins/rapture/nexus-rapture-baseapp/src/main/baseapp/packages/rapture-theme/sass/src/Ext/button/Button.scss [hunk 2]
```diff
@@ -84,7 +84,7 @@
   $inner-opacity-disabled: 1
 );
 
-.x-btn-drilldown-medium {
+.x-btn-ux-drilldown-medium {
```
Comment (adreghiciu) [inline plugins/rapture/nexus-rapture-baseapp/src/main/baseapp/packages/rapture-theme/sass/src/Ext/button/Button.scss]: ux?

File: plugins/rapture/nexus-rapture-baseapp/src/main/resources/static/rapture/resources/baseapp-debug_01.css [hunk 3]
```diff
@@ -23910,339 +23910,339 @@ a.x-color-picker-item:hover {
...
   display: none;
-  content: "x-slicer:stretch:bottom, corners:url(images/btn/btn-drilldown-large-disabled-corners.gif), sides:url(images/btn/btn-drilldown-large-disabled-sides.gif), frame-bg:url(images/btn/btn-drilldown-large-disabled-fbg.gif), bg:url(images/btn/btn-drilldown-large-disabled-bg.gif)";
+  content: "x-slicer:stretch:bottom, corners:url(images/btn/btn-nx-drilldown-large-disabled-corners.gif), sides:url(images/btn/btn-nx-drilldown-large-disabled-sides.gif), frame-bg:url(images/btn/btn-nx-drilldown-large-disabled-fbg.gif), bg:url(images/btn/btn-nx-drilldown-large-disabled-bg.gif)";
 }
 
 /*</if slicer>*/
 /* */
 /* line 46, ../../../baseapp/packages/rapture-theme/sass/src/Ext/button/Button.scss */
-.x-btn-drilldown-large {
+.x-btn-ux-drilldown-large {
```
Comment (adreghiciu) [inline plugins/rapture/nexus-rapture-baseapp/src/main/resources/static/rapture/resources/baseapp-debug_01.css]: ux?

**Comment by adreghiciu** [comment]
+1

File: plugins/rapture/nexus-rapture-baseapp/src/main/baseapp/packages/rapture-theme/sass/src/Ext/button/Button.scss (hunk 1 above)
Comment (kellyrob99) [inline plugins/rapture/nexus-rapture-baseapp/src/main/baseapp/packages/rapture-theme/sass/src/Ext/button/Button.scss]: also, why not prefix here as elsewhere?

File: plugins/rapture/nexus-rapture-baseapp/src/main/baseapp/packages/rapture-theme/sass/src/Ext/button/Button.scss [hunk 4]
```diff
@@ -92,12 +92,12 @@
   }
 }
 
-.feature-content .x-panel-header .x-btn-drilldown-medium {
+.nx-feature-content .x-panel-header .x-btn-nx-drilldown-medium {
```
Comment (kellyrob99) [inline plugins/rapture/nexus-rapture-baseapp/src/main/baseapp/packages/rapture-theme/sass/src/Ext/button/Button.scss]: Not sure why we would deviate from prefixing with nx to embedding in the middle... is this an ExtJS thing?

Comment (djsauble) [inline plugins/rapture/nexus-rapture-baseapp/src/main/baseapp/packages/rapture-theme/sass/src/Ext/button/Button.scss]: Yes, it’s an extjs thing.

The value of 'ui' propagates to child classes. In the above example, 'ui' has been set to 'nx-drilldown', which turns .x-btn-{ui}-drilldown-medium into .x-btn-nx-drilldown-medium.

**Comment by djsauble** [comment]
Manual merge.

File: plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/dev/styles/Header.js [hunk 5]
```diff
@@ -91,26 +91,26 @@ Ext.define('NX.view.dev.styles.Header', {
           '->',
           //{
           //  xtype: 'button',
-          //  ui: 'header',
+          //  ui: 'nx-header',
```
Comment (kellyrob99) [inline plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/dev/styles/Header.js]: would rather delete the commented out code than update ;)

File: plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/dev/styles/Fonts.js [hunk 6]
```diff
@@ -90,48 +90,48 @@ Ext.define('NX.view.dev.styles.Fonts', {
...
+        ui: 'nx-subsection',
         layout: {
           type: 'hbox',
           defaultMargins: {top: 0, right: 20, bottom: 0, left: 0}
         },
         items: [
-          faceExample('Proxima Nova Regular', 'proxima-nova-regular'),
-          faceExample('Proxima Nova Bold', 'proxima-nova-bold'),
-          faceExample('Courier New', 'courier-new-regular')
+          faceExample('Proxima Nova Regular', 'nx-proxima-nova-regular'),
```
Comment (kellyrob99) [inline plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/dev/styles/Fonts.js]: not sure if we want to do this, these are not really 'our' fonts but don't feel strongly either way

**Comment by kellyrob99** [comment]
+1

**Pull request closed without merging**